tools/linter_lib/custom_check.py

```diff
@@ -870,6 +870,20 @@
             "description": "Likely missing space after end of sentence",
             "include_only": {"help/"},
             "exclude_pattern": "Rocket.Chat",
```

Comment:
I think it would be cleaner to add `org.zulip.Zulip` to the `"exclude_pattern"`, and drop the whole `"exclude_line"` part.

```suggestion
            "exclude_pattern": "Rocket.Chat|org.zulip.Zulip",
```